### ISSUE TITLE
Question: does this rate limit by IP or is it just by caller?

### ISSUE THREAD

**Issue by Integralist** [tomasbasham/ratelimit#30]
As per the title, I just wanted to check if this rate limits by IP or by caller?

Couldn't see anything related to IP restrictions in the code so am presuming it's a fairly simple implementation.

Cool project nonetheless though, so thanks for sharing it with the world 👍

**Comment by tomasbasham** [comment]
The name of the package may be misleading as it is more akin to throttling. When I originally wrote it I was working with making API requests which led me to the name.

**Comment by konstunn** [comment]
Yes. Shortly speaking, this project is about clients, not servers.
